.eslintrc

```diff
@@ -13,6 +13,8 @@
     ],
     "rules": {
       "no-console": 1,       // Means warning
-      "prettier/prettier": 2 // Means error
+      "prettier/prettier": 2,
+
+      "@typescript-eslint/no-inferrable-types": "off"
```

Comment:
what is this one doing?

Reply:
this one was conflicting a lot it's when you set `foo: boolean = true` this rule enforce that your remove `: boolean` because some primitives types are inferable. It's debatable ^^